README.md

```diff
@@ -46,14 +32,7 @@ We first need to start a local Espresso Sequencer network.
     docker compose pull
     docker compose up -d --wait
 
-Once that is finished deploy the example rollup, it can be run natively or with docker.
-
-To run the demo inside docker run the following
-
-    docker compose pull
-    docker compose up
-
-To stop the docker demo run `docker compose down`.
+Once that is finished deploy the example rollup, it can be run natively.
 
 To run natively a rust toolchain is required. We provide a nix shell to install
 all dependencies. Nix can be installed via the
```

Comment:
For the section below with `just dev-demo` make it clear which folder the user needs to be in

Reply:
https://github.com/EspressoSystems/sequencer-example-l2/pull/71/commits/9c85050f35e37c3dde45993b36089da11edafab4